build/common.props

```diff
@@ -34,9 +34,19 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <PackageValidationBaselineVersion>6.33.0</PackageValidationBaselineVersion>
+  </PropertyGroup>
+
   <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);SYSLIB0050</NoWarn>
```

Comment:
Do we have a follow up task to deal with these warnings If we suppress now? or is that not a concern?

Reply:
It's something that has to be kept to maintain support for older target frameworks. Such a task could come up when we stop supporting whichever tf needs it.

Reply:
Sounds good, resolving this comment.